test/verify/check-storage-resize

```diff
@@ -93,6 +93,12 @@ class TestStorageResize(StorageCase):
         else:
             self.wait_content_tab_action_disabled(1, 1, "Grow")
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1934567
```

Comment:
Wouldn't that be better as a naughty?

Reply:
The naughty will make the test fail premature (xfail of course). With this small Workaround we make sure that the test can finish, and that the rest of functionality is actually ok.
 
If you disagree I am happy to extend the existing naughty.

Lastly, when this issue is fixed, the Mount button will be missing so this will start failing. In this way we will get naughtified and we will revert the HACK.

Reply:
I will create a naughty

Reply:
If it skips too much functionality which is otherwise working, that's ok -- I mostly was interested in the rationale. Thanks!

Reply:
https://github.com/cockpit-project/bots/pull/1746 contains already the naughty.